core-services/prow/02_config/opendatahub-io/odh-dashboard/_prowconfig.yaml

```diff
@@ -0,0 +1,117 @@
+tide:
+  queries:
+  - includedBranches:
+    - community-4.6
+    - community-4.7
+    - openshift-4.10
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.0
+    - release-4.10
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
```

Comment:
For now I think we only want:

```
- main
``` 

See https://github.com/openshift/release/blob/master/core-services/prow/02_config/red-hat-data-services/odh-deployer/_prowconfig.yaml

Reply:
Thx. Updated